feature-utils/poly-look/src/react-components/contexts/poly-import.jsx

```diff
@@ -38,7 +38,7 @@ export const PolyImportProvider = ({
           return;
         }
         for (const file of storage.files) {
-          resolvedFiles.push(await file);
+          resolvedFiles.push(file);
```

Comment:
It's called `resolvedFiles`, and the `await` is what _resolves_ them. Are you sure this can go?

That said, a more succinct way of doing this would be:

`const resolvedFiles = await Promise.all(storage.files);`

Reply:
OK, `stat` returns a `Promise<Stats>`. That should have been obvious. Still, naming matters...

Reply:
I guess `pendingFiles` would work, though I personally find it obvious enough.